teuthology/task/internal.py

```diff
@@ -342,6 +348,50 @@ def push():
     except Exception:
         log.exception("Error pushing inventory")
 
+BUILDPACKAGES_SWAPPED = 0
+BUILDPACKAGES_OK = 1
+BUILDPACKAGES_REMOVED = 2
+BUILDPACKAGES_NOTHING = 3
+
+def buildpackages(ctx, config):
```

Comment:
I was massively confused about the difference between ``buildpackages`` and ``internal.buildbackages``. Would it make sense perhaps to rename the latter to ``internal.buildpackages_prep`` or something similar?